examples/experimental/sun/src/shadow-effect/shadow-module.js

```diff
@@ -29,7 +29,6 @@ varying vec3 shadow_vPosition;
 
 const vec4 bitPackShift = vec4(1.0, 256.0, 256.0 * 256.0, 256.0 * 256.0 * 256.0);
 const vec4 bitUnpackShift = 1.0 / bitPackShift;
-const vec4 bitMask = vec4(1.0 / 256.0, 1.0 / 256.0, 1.0 / 256.0,  0.0);
```

Comment:
I believe that there is a `pack` shader module somewhere that does this. @1chandu might remember the details.